CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 Changelog
 =========
 
+## 0.5.6
```

Comment:
It's fine for now (of course you couldn't know), but let's use `## vNext` for the "next unknown version" of the plugin in the changelog section 😉  otherwise it seems like we already released that version!